server/tests/load_postgres_data.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import getpass
+import os
+import subprocess
+
+conf = {
+    "staging": {
+        "port": 5432,
+    },
+    "production": {
+        "port": 5433,
+    },
+}
+
+
+def run_pg_cmd_on_guest(guest_cmd, password):
+    """Run a postgres command in the guest container."""
+    return subprocess.run([
+        "docker-compose exec -e PGPASSWORD=%s postgres bash -c '%s'" % (
+            password, guest_cmd),
+    ], shell=True, check=True)
+
+
+def parse_args():
+    """Parse arguments for the posgres data loader command."""
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("environment", default="staging",
+                        help="Name of the environment the data will be taken from")
+    parser.add_argument("--remote-postgres-user",
+                        default=os.environ.get("CLOUD_SQL_POSTGRES_USER"),
+                        help="Name of the user to use for the local db")
+    parser.add_argument("--local-postgres-user",
+                        default=os.environ.get("POSTGRES_USER", "api"),
+                        help="Name of the user to use for the local db")
+
+    parsed_args = parser.parse_args()
+
+    if parsed_args.environment not in conf:
+        raise ValueError("Unknown environment")
+
+    if not parsed_args.remote_postgres_user:
+        raise ValueError("Missing remote postgres user")
+
+    if not parsed_args.local_postgres_user:
+        raise ValueError("Missing local postgres user")
+
+    return parsed_args
+
+
+def main():
+    """Run the postgres data loader."""
+    try:
+        args = parse_args()
+    except ValueError:
+        return 1
```

Comment:
```suggestion
    except ValueError as e:
        print(e, file=sys.stderr)
        return 1
```

Otherwise, the user will not see the error message.